source/.pylintrc

```diff
@@ -7,7 +7,7 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=Secrets,unit_tests,OptimizerService_pb2.py,OptimizerService_pb2_grpc.py
+ignore=Secrets,unit_tests,OptimizerService_pb2.py,OptimizerService_pb2_grpc.py,long_haul_tests,DistributableSimpleBayesianOptimizer.py
```

Comment:
Isn't that deleted? I thought SimpleBayesianOptimizer went away?

Reply:
It had gone away in ADO, but Brian just yesterday started moving the changes over to Github. Once his PR is done I'll rebase and remove it from here.

Reply:
It's done now.